src/tm2/tm-boot.h

```diff
@@ -22,21 +22,22 @@ namespace librealsense {
                     LOG_INFO("Found a T265 to boot");
                     found = true;
                     auto dev = usb_enumerator::create_usb_device(device_info);
-                    const auto& m = dev->open(0);
+                     if (const auto& m = dev->open(0))
```

Comment:
No message/error if it failed?

Reply:
In this case the error is reported by the invoking code